plugins/jira/README.md

```diff
@@ -18,11 +18,11 @@ This plugin supplies one command, `jira`, through which all its features are exp
 jira            # performs the default action
 
 jira new        # opens a new issue
+jira mine       # queries for you own issues
```

Comment:
`your` instead of `you`

Reply:
thanks! I just amended the commit and force pushed, hope that's alright.